scripts/train_agents.py

```diff
@@ -153,12 +153,18 @@ def set_input(args, quick_test=False):
     args.layout_names = ['3_chefs_small_kitchen']
     args.teammates_len = 2
     args.num_players = args.teammates_len + 1  # 3 players = 1 agent + 2 teammates
-    args.learner_type = LearnerType.SUPPORTER
     
+    if supporter_run:
+        args.learner_type = LearnerType.SUPPORTER
+        args.reward_magnifier = 3.0
+    else:
+        args.learner_type = LearnerType.SOLOWORKER
+        args.reward_magnifier = 1.0
+        
     if not quick_test: 
-        args.n_envs = 50
+        args.n_envs = 200
+        how_long = 4.0
```

Comment:
Do we want the default value for `how_long` to be 4.0? I think we should leave it at 1.0

Reply:
fair enough!